ugandaemr_mobile/assess-patients.md

```diff
@@ -0,0 +1,16 @@
+# Work Summary
```

Comment:
@solemabrothers I think you should combine, Download Patients, Find Patients, Work Summary, Drug Dispensing into a single file - I do not see sufficient content for 3 pages

Reply:
@solemabrothers The first heading should always be a heading 2 (##) not heading 1

Reply:
Done